src/multimediawidgets/CMakeLists.txt

```
@@ -32,24 +32,40 @@ set_target_properties(${BIN_NAME} PROPERTIES
 
 target_compile_definitions(${BIN_NAME} PRIVATE VERSION="${CMAKE_PROJECT_VERSION}")
 
-find_package(Qt5Widgets)
-find_package(Qt5DBus)
-find_package(Qt5X11Extras)
-find_package(Qt5LinguistTools)
-find_package(Qt5Network)
-find_package(Qt5Concurrent)
-find_package(Qt5Multimedia)
-find_package(Qt5MultimediaWidgets)
+find_package(Qt${QT_VERSION_MAJOR}Widgets)
+find_package(Qt${QT_VERSION_MAJOR}DBus)
+find_package(Qt${QT_VERSION_MAJOR}LinguistTools)
+find_package(Qt${QT_VERSION_MAJOR}Network)
+find_package(Qt${QT_VERSION_MAJOR}Concurrent)
+find_package(Qt${QT_VERSION_MAJOR}Multimedia)
+find_package(Qt${QT_VERSION_MAJOR}MultimediaWidgets)
+if (ENABLE_Qt6)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS OpenGLWidgets)
+endif()
 find_package(PkgConfig REQUIRED)
 
-pkg_check_modules(Dtk REQUIRED IMPORTED_TARGET dtkwidget)
-pkg_check_modules(Dtk REQUIRED IMPORTED_TARGET dtkcore)
-pkg_check_modules(DBusextended REQUIRED IMPORTED_TARGET dbusextended-qt5)
+if (ENABLE_Qt6)
+    message("find_package................")
+    find_package(Dtk${QT_VERSION_MAJOR}Core)
+    find_package(Dtk${QT_VERSION_MAJOR}Widget)
+#    find_package(Qt6 COMPONENTS Core5Compat REQUIRED)
+else()
+    pkg_check_modules(Dtk REQUIRED IMPORTED_TARGET dtkwidget)
+    pkg_check_modules(Dtk REQUIRED IMPORTED_TARGET dtkcore)
+    pkg_check_modules(DBusextended REQUIRED IMPORTED_TARGET dbusextended-qt5)
+endif()
```

```suggestion
if (ENABLE_Qt6)
    find_package(Dtk${QT_VERSION_MAJOR}Core)
    find_package(Dtk${QT_VERSION_MAJOR}Widget)
else()
    pkg_check_modules(DBusextended REQUIRED IMPORTED_TARGET dbusextended-qt5)
endif()
```
dtkcore/dtkwidget不需要区分，已经适配过 qt6 了。 虽然还没有打包。
调试信息和无效的注释删掉